### DISCUSSION TITLE
How to add custom element to CartesianChart?

### DISCUSSION THREAD

**Discussion by Martin-Yuan** [Live-Charts/LiveCharts2#713] [Q&A]
I wanna show the point value next to the  intersection point of line and section .  
Any way to make it ?

**Comment by Martin-Yuan** [comment]
I found the VisualElements demo in livechart(v2) , and it doesn't work. 

frontend codes:
  <lvc:CartesianChart x:Name="lvChart" ZoomMode="X" Grid.Row="1" VisualElements="{Binding VisualElements}"
            Series="{Binding Series}"
            XAxes="{Binding XAxes}" Sections="{Binding  Sections}">
        </lvc:CartesianChart>


backend codes:
public ObservableCollection<ChartElement<SkiaSharpDrawingContext>> VisualElements { get; set; }
 var item = dataPoints[idx];
            var label = new LabelVisual
            {
                Text = $"Data:{item.Value}",
                X = item.DateTime.Ticks,
                Y = item.Value.Value,
                TextSize = 16,
                Paint = new SolidColorPaint(SKColors.Red) { ZIndex = 11 },
               // BackgroundColor = new LvcColor(55, 71, 79),
                Padding = new Padding(12),
                LocationUnit = MeasureUnit.ChartValues,
                //Translate = new LvcPoint(0, -35)
            };

            VisualElements.Add(label);

when "LocationUnit =MeasureUnit.Pixels"  is working well. so strange???

**Comment by GF-Huang** [comment]
I have the same question, but can you improve your code to make it easier to read?

\```xaml
// your xaml code
\```

\```cs
// your C# code
\```
